### PR TITLE
[ansible] add testing for debian 12

### PR DESCRIPTION
**Pull request by atoulme**
**Description:**
Add testing of debian 12 as part of the test matrix of our ansible kit.

### CODE REVIEW

**Comment by atoulme** [comment]
Closing for now as we will need to decide how to support fluentd on debian bookworm.

File: deployments/ansible/molecule/shared/verify_tasks.yml [hunk 1]
```diff
@@ -11,6 +11,8 @@
         false
       {%- elif ansible_distribution == "Amazon" and ansible_distribution_version == "2023" -%}
         false
+      {%- elif ansible_os_family == "Debian" and ansible_distribution_major_version == "12" -%}
```
Comment (pjanotti) [inline deployments/ansible/molecule/shared/verify_tasks.yml:14]: Is fluentd is supported on debian prior to 12?

**Review by pjanotti — APPROVED** [review]
LGTM, but, one Q about fluentd support.

The Ansible for Windows test failures are unrelated to this change.

File: deployments/ansible/molecule/shared/verify_tasks.yml (hunk 1 above)
Comment (atoulme) [reply deployments/ansible/molecule/shared/verify_tasks.yml:14]: it's not supported by the rpm provider.

**Review by pjanotti — APPROVED** [review]

**Pull request merged**